pkg/graph/resolvers/it_gov_task_statuses_test.go

```diff
@@ -666,13 +673,214 @@ func (suite *ResolverSuite) TestBizCaseFinalStatus() {
 	suite.EqualValues(models.ITGFBCSCantStart, status)
 
 }
-func (suite *ResolverSuite) TestGrbMeetingStatus() {
-	intake := models.SystemIntake{
-		Status: models.SystemIntakeStatusCLOSED,
-	}
+func TestGrbMeetingStatus(t *testing.T) {
 
-	status := GrbMeetingStatus(&intake)
+	yesterday := time.Now().Add(time.Hour * -24)
+	tomorrow := time.Now().Add(time.Hour * 24)
+	invalidTestStep := models.SystemIntakeStep("Testing Invalid Step")
 
-	suite.EqualValues(models.ITGGRBSCantStart, status)
+	grbTests := []testSystemIntakeGRBStatusType{
+		{
+			testCase: "Request form: No GRB Date Scheduled",
```

Comment:
The only thing I don't like about this is it feels like you could do a nested loop to generate these test cases instead of a giant copy/paste. It's not the worst thing and this is a nitpick, but it does make it a little tougher to spot missing test cases when it's a giant file.

Reply:
Yeah, I hear you there. I don't really like that this makes the file so long either, but I do like being explicit about the test case 🤷. I don't have a solution I love to change this to right now, but I'd definitely be up for a different approach. Maybe we could do a pairing sometime to brainstorm this? 

I thought about keeping test cases in a separate file, or perhaps representing as a JSON or CSV and import it, but I decided against it.